build.gradle.kts

```diff
@@ -56,7 +56,7 @@ dependencies {
         exclude(module = "fabric-gametest-api-v1")
     }
     modImplementation("dev.gegy:markdown-chat:1.3.0", excludeFabricApi)
-    modImplementation("com.github.samolego.Config2Brigadier:config2brigadier-fabric:1.2.1", excludeFabricApi)
+    // modImplementation("com.github.samolego.Config2Brigadier:config2brigadier-fabric:1.2.1", excludeFabricApi)
```

Comment:
Out of scope change; may consider mirroring this instead of this was to work around Jitpack-related bugs.